app/src/main/res/xml/pref_data_sync.xml

```diff
@@ -7,7 +7,7 @@
         android:key="dataSync"
         android:title="@string/data_sync">
         <PreferenceScreen
-            android:icon="@drawable/ic_scan"
+            android:icon="@drawable/ic_qrscan"
```

Comment:
The barcode scan icon is present in screenshot documentation and as mentioned in another comment I don't think the auto-configuration feature is limited to QR codes. From an aesthetic point of view the red coloring is the only highlight color on the settings top level which matches the xDrip drop icon.

Reply:
> From an aesthetic point of view the red coloring is the only highlight color on the settings top level which matches the xDrip drop icon.

Shall I change the color of the edge bars from green to red? Removing it all together makes the icon indistinguishable to the "Show Settings QR codes" icon.

Reply:
@tolot27 What if you were to change it to red, _and_ add a horizontal line across the middle, similar to this:
![image](https://user-images.githubusercontent.com/52087143/156068362-ff08e02a-7981-4f13-8bbb-ebe4bec84641.png)

That would address both of @jamorham's concerns, since the horizontal line implies barcode support. I also found these icons as well, which we could potentially use as a reference for a new icon:
![image](https://user-images.githubusercontent.com/52087143/156069975-6211fbe2-a6e8-496e-bf50-089bacd843db.png)
![image](https://user-images.githubusercontent.com/52087143/156069840-bce94367-4843-43c8-a215-dfd562a18ae5.png)

Reply:
I've moved the QR icon to #2022.

Reply:
@Lupum1001 Thanks for your suggestions. Let us continue this discussion at #2022.